src/DataTable/TablePaginationMinimal.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { Pagination } from '..';
+import DataTableContext from './DataTableContext';
+
+const TablePaginationMinimal = () => {
+  const {
+    nextPage, pageCount, gotoPage, state,
+  } = useContext(DataTableContext);
+
+  // Use nextPage as a proxy for whether or not the table is paginated
+  if (!nextPage) {
+    return null;
+  }
+
+  const pageIndex = state?.pageIndex;
+
+  return (
+    <Pagination
+      variant="minimal"
+      currentPage={pageIndex + 1}
+      pageCount={pageCount}
+      paginationLabel="datatable pagination"
```

Comment:
nit: just "table pagination" should be fine here; don't think users need to know its a DataTable specifically.

Reply:
fixed